bindings/experimental/distrdf/python/DistRDF/Backends/Utils.py

```diff
@@ -134,25 +136,93 @@ def check_pcm_in_library_path(shared_library_path: str) -> Tuple[Set[str], Set[s
         list, list: Two lists, the first with all paths to pcm files, the
             second with all paths to shared libraries.
     """
+    
+    # issue when we have, for example ROOT.RDF.Experimental.Distributed.DistributeSharedLibs("folder/helpers_h.so") - this is False and should be True 
+    # therefore this will not work either and instead we check for the file endings of possible shared libraries 
+    # if os.path.isfile(shared_library_path):
+    #     shared_library_dir = os.path.dirname(os.path.abspath(shared_library_path))
+    #     #shared_library_dir = os.path.abspath(shared_library_path)
+    #     #+ "/"
+    
+    shared_library_formats = ['.so', '.dll', '.dylib']
+    
+    if shared_library_path.endswith(tuple(shared_library_formats)): 
```

Comment:
```suggestion
    shared_library_formats = ('.so', '.dll', '.dylib')
    
    if shared_library_path.endswith(shared_library_formats): 
```
This should remove the need for the extra conversion